src/main/definitions/case.ts

```diff
@@ -32,8 +32,12 @@ export interface Case {
   presentEmployer?: YesOrNo;
   typeOfClaim?: TypesOfClaim[];
   pastEmployer?: YesOrNo;
+  noticeEnd?: string;
   noticePeriod?: YesOrNo;
   noticePeriodLength?: string;
+  noticePeriodUnit2?: WeeksOrMonths;
```

Comment:
I don't like the names of these variables with 2 at the end. They aren't very descriptive. Please make sure you aren't duplicating values that already exist like noticePeriodLength. It looks like that value could probably be refactored to be WeeksOrMonths and shared between the 'working' and 'notice period' flows.